dns/rfc2136/src/etc/inc/plugins.inc.d/rfc2136.inc

```diff
@@ -74,7 +74,7 @@ function rfc2136_cron()
     $jobs = [];
 
     if (rfc2136_enabled()) {
-        $jobs[]['autocron'] = ['/usr/local/etc/rc.rfc2136', '16', '1'];
+        $jobs[]['autocron'] = ['/usr/local/etc/rc.rfc2136', '*/5'];
```

Comment:
Care to explain this change? It’s not very useful to run the service twice every five minutes vs. once a day. The DNS change does have a dynamic trigger… so we don’t need excessive polling.

Reply:
So you are saying that if the IP address changes it will automatically trigger this plugin to perform the DNS update?

Reply:
Reverted this change in https://github.com/opnsense/plugins/pull/3316/commits/c8ab1c3ee3222a6c0fc6808b3e91afa8367b2215

Reply:
yes, from core:

```
% git grep rfc2136
src/etc/inc/interfaces.inc:        configdp_run('rfc2136 reload', [$interface]);
```

and from plugins:

```
% git grep rfc2136_configure_do
src/etc/inc/plugins.inc.d/rfc2136.inc:        'bootup' => ['rfc2136_configure_do'],
src/etc/inc/plugins.inc.d/rfc2136.inc:        'local' => ['rfc2136_configure_do'],
src/etc/inc/plugins.inc.d/rfc2136.inc:        'newwanip' => ['rfc2136_configure_do:2'],
```

Reply:
Thanks, I was unaware. This change was reverted so I think this is resolved now?

Reply:
Yep :)